src/lib/setup.ts

```diff
@@ -0,0 +1,23 @@
+import ddTracer from "dd-trace"
+
+export const setupDataDog = () => {
+  if (process.env.DD_APM_ENABLED) {
+    ddTracer.init({
+      hostname: process.env.DD_TRACE_AGENT_HOSTNAME,
+      service: "kaws",
+      plugins: false,
+    })
+    ddTracer.use("express", {
+      // We want the root spans of MP to be labelled as just `service`
```

Comment:
This comment and the one below are possibly copied from Metaphysics.

Reply:
That would make sense. I wasn't sure if they were relevant. I'll remove them.